chat/agent.js

```diff
@@ -840,7 +841,29 @@ window.menu = null;
                     var n = new RegExp("(?:^|\\s)" + e.CLASS_SEND_ACTIVE + "(?!\\S)", "gi");
                     return this.sendBtn.className = this.sendBtn.className.replace(n, ""), this
                 }
-            }, {
+            }, { 
+              key: "addStopStreamingButton",
+              value: function() {
+                if (!this.sendBtn.classList.contains(e.CLASS_SEND_ACTIVE)) {
+                  // remove send icon
+                  this.sendBtn.classList.remove('icon-send');
+                  this.sendBtn.className += " " + e.CLASS_SEND_ACTIVE;
+                }
+
+                var stopIcon = document.getElementById("icon-stop");
+                if (stopIcon) {
+                  return this.sendBtn, this
+                } else {
+                  this.sendBtn.innerHTML = `
+                    <svg id="icon-stop" xmlns="http://www.w3.org/2000/svg" version="1.1" width="32" height="32" viewBox="0 0 32 32">
+                      <title>stop</title>
+                      <path fill="red" d="M16 0c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zM16 29c-7.18 0-13-5.82-13-13s5.82-13 13-13 13 5.82 13 13-5.82 13-13 13zM10 10h12v12h-12z"/>
```

Comment:
button background variable is now:

var(--button-background)

Reply:
please double check `var(--button-background)`
I added : `<path fill="var(--button-background)",` but I didn't see a `button-background` var.